js/scripts/zone_loaders/playground_zone_loaders.js

```diff
@@ -216,15 +216,10 @@ const loadPlaygroundNathan = () => {
         CANVAS.addEventListener('dblclick', function(e) {
             e.preventDefault();
         });
-
-        loadingAnimation.stop(); // stop the loading animation because asset manager has everything it needs
     };
 
     // Set background color:
-    BG_COLOR = "skyblue";
-
-    let loadingAnimation = new LoadingAnimation();
```

Comment:
Are we still using this? It might be cool to have a loading animation. I would love to have something like that between zones.

Reply:
We are technically using a LoadingAnimation, however, it hasn't shown up yet. When I have more time, I'll debug it and ensure it's being played at the correct time.